storage/time-series/influxdb/retriever/app/express/interceptors/DojotTenantJwtParse.js

```diff
@@ -0,0 +1,39 @@
+const createError = require('http-errors');
+const jwtDecode = require('jwt-decode');
+
+/* *
+ * A middleware  to  extract dojot tenant from jwt
+ *
+ * Decodes the JWT token that must be sent with the request.
+ * The token validation is not performed, as it is expected
+ * to be validated by the API Gateway.
+ * And add in request the new tenant (req.tenant)
+ *
+ * */
+module.exports = () => ({
```

Comment:
Wouldn't be a candidate to be in the sdk?

Reply:
@thiagodpf

Reply:
It would be, but I thought that with the adoption of the keycloak, things would change, so I didn't put in the PR that I did for the sdk.